dali/python/nvidia/dali/ops.py

```diff
@@ -773,14 +774,18 @@ def _wrap_op(op_class, submodule=[], parent_module=None):
                                 _docstring_generator_fn(op_class))
 
 
-def _load_ops():
+def _get_all_registered_ops():
     global _cpu_ops
     global _gpu_ops
     global _mixed_ops
     _cpu_ops = _cpu_ops.union(set(_b.RegisteredCPUOps()))
     _gpu_ops = _gpu_ops.union(set(_b.RegisteredGPUOps()))
     _mixed_ops = _mixed_ops.union(set(_b.RegisteredMixedOps()))
```

Comment:
Now that I look at it - why is it a union? Why not just replace? Can something disappear from `_b.RegisteredXXXOps`?

Reply:
:man_shrugging: